infra/charts/turing-init/README.md

```diff
@@ -20,8 +20,10 @@ A Helm chart for Kubernetes
 | image.tag | string | `"latest"` | Docker image tag for Turing cluster init |
 | istioOperatorConfig | object | `{"apiVersion":"install.istio.io/v1alpha1","kind":"IstioOperator","spec":{"addonComponents":{"pilot":{"enabled":true}},"components":{"ingressGateways":[{"enabled":true,"name":"istio-ingressgateway"},{"enabled":true,"k8s":{"service":{"ports":[{"name":"status-port","port":15020},{"name":"http2","port":80},{"name":"https","port":443}],"type":"ClusterIP"}},"label":{"app":"cluster-local-gateway","istio":"cluster-local-gateway"},"name":"cluster-local-gateway"}]},"values":{"gateways":{"istio-ingressgateway":{"runAsRoot":true}},"global":{"proxy":{"autoInject":"enabled"},"useMCP":false}}}}` | istio operator config, defaults are the minimum to run turing |
 | istioVersion | string | `"1.9.9"` | Istio version to use |
-| knativeIstioVersion | string | `"0.18.1"` |  |
+| knativeDomains | string | `""` | Knative domains, comma seperated values, i.e. www.example.com,www.gojek.com |
+| knativeIstioVersion | string | `"0.18.1"` | Knative Istio Version to use |
 | knativeVersion | string | `"0.18.3"` | Knative Version to use |
+| knative_registries_skipping_tag_resolving | string | `""` | Knative registries skipping tag resolving, comma seperated values, i.e. www.example.com,www.gojek.com |
```

Comment:
Can we have a better structure of the values file by grouping knative/istio properties under the root-level parent config groups:
```yaml
image:
  tag: ...

istio:
  version: ...
  operatorConfig: ...

knative:
  version: ...
  domains: ...
  istioVersion: ...
  registriesSkippingTagResolving: ...

spark-operator: 
  image: ...
  
```

Reply:
Why is it snake here, shouldn't it be a camelCase instead?

Reply:
> Why is it snake here, shouldn't it be a camelCase instead?

Should have been regenerated, its supposed to be camel case, i'll regenerate this after i do the changes.